youtube_dl/extractor/adobepass.py

```diff
@@ -20,6 +20,11 @@
 
 
 MSO_INFO = {
+    'ATT': {
+        'name': 'AT&T U-Verse',
+        'username_field': 'nameBox',
+        'password_field': 'pwdBox'
```

Comment:
Browser does not send such fields.

Reply:
It sends "userid" and "password" instead.
`<input type="text" name="userid" id="nameBox" value="" maxlength="50" size="50" class="slidTxtbox" title="Userid" style="color: rgb(170, 170, 170);">`
`<input type="text" placeholder="Password" name="password" id="pwdBox" value="" maxlength="50" size="50" class="slidTxtbox" autocomplete="off" title="Password">`